tests/common/test_event.py

```diff
@@ -163,6 +165,23 @@ def test_save_event(self):
         add_event('test', message='test event')
         self.assertTrue(len(os.listdir(self.tmp_dir)) == 2)
 
+    def test_save_event_message_with_non_ascii_characters(self):
+        stdout = open(os.path.join(data_dir, "events", "collect_and_send_extension_stdout_stderror",
+                                   "dummy_stdout_with_non_ascii_characters"), mode="r+b")
+        stderr = open(os.path.join(data_dir, "events", "collect_and_send_extension_stdout_stderror",
+                                   "dummy_stderr_with_non_ascii_characters"), mode="r+b")
+
+        msg = read_output(stdout, stderr)
+        duration = elapsed_milliseconds(datetime.utcnow())
+        log_msg = "{0}\n{1}".format("DummyCmd", "\n".join([line for line in msg.split('\n') if line != ""]))
+
+        add_event('test_extension', message=log_msg, duration=duration)
+        for tld_file in os.listdir(self.tmp_dir):
+            event_str = MonitorHandler.collect_event(os.path.join(self.tmp_dir, tld_file))
+            event_json = json.loads(event_str)
+
+            self.assertEqual(8, len(event_json["parameters"]))
```

Comment:
can we have a stronger assertion? e.g. all the expected parameters (Name, etc) are present?